tailcfg/tailcfg.go

```diff
@@ -2321,6 +2322,11 @@ const (
 	// We began creating this rule on 2024-06-14, and this node attribute
 	// allows us to disable the new behavior remotely if needed.
 	NodeAttrDisableLocalDNSOverrideViaNRPT NodeCapability = "disable-local-dns-override-via-nrpt"
+
+	// NodeAttrCaptivePortalDetection instructs the client to perform captive portal detection
+	// automatically when the network state changes, putting the associated Warnable in an unhealthy
+	// state if a captive portal is detected.
+	NodeAttrCaptivePortalDetection NodeCapability = "captive-portal-detection"
```

Comment:
I don't want these to pile up forever with the server having to tell the client to do everything. If this is going to start opt-in and later move to opt-out, talk about that schedule here and link to a tracking bug to remove this nodeAttr? Or I guess we can't ever remove it, since we use it server-side for old clients. So we need both a force-on and force-off variant.

Reply:
Changed it to exclusively be an opt-out flag.